master/getting-started/docker/installation/prerequisites.md

```diff
@@ -0,0 +1,39 @@
+---
+title:  Calico-Docker Prerequisites
+---
+
+Calico networking for docker containers requires that all docker hosts
+have IP connectivity between them.
+
+You will need at least one server (bare metal or VM) with a modern 64-bit Linux OS
+and IP connectivity between them.
+
+### etcd
+
+You will also need an etcd cluster accessible from each host which Calico
+uses for coordinating state between the nodes. See the [etcd documentation][etcd]
+for details on setting up a cluster.
+
+### Docker with Multi-Host networking
```

Comment:
General problem with our headings ... capitalize each first word or not... I say not.